runtime/runtime-params-estimator/src/main.rs

```diff
@@ -114,7 +114,10 @@ fn main() -> anyhow::Result<()> {
             );
 
             let near_config = load_config(&state_dump_path);
-            let store = create_store(&get_store_path(&state_dump_path));
+            let store = create_store_with_extra_cache(
+                &get_store_path(&state_dump_path),
+                near_config.config.extra_db_memory_bytes,
```

Comment:
This should be `0` -- we are interested in estimating worst-case, when the cache is not used. Though, I guess we'll get `0` here as that's the default for config anyway? 

I guess, it's fine as is, though I'd slightly perfer explicit `0`.